### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by ailisp** [near/near-sdk-js#351] [Announcements]
## Highlights
- [Fungible Token Standard and Storage Management](https://github.com/near/near-sdk-js/tree/develop/packages/near-contract-standards) are added to near-contract-standards
- [Tests to ensure arithmentic correctness of quickjs runtime](https://github.com/near/libbf-test) are finished. Because of this, we move forward from alpha to release!
- [Reference Doc](https://docs.near.org/sdk/near-sdk-js/reference) is now available!
- Contribution guideline is added to the repo!
- [A gas and storage benchmark is added](https://github.com/near/near-sdk-js/tree/develop/benchmark)
- Enumerate keys/elements in a pagination way is added in collections API.
- Several bug fixes in build tools.

## What's Changed
* Fix typo in rollup command name by @idea404 in https://github.com/near/near-sdk-js/pull/336
* Enumerate keys/elements and pagination by @ailisp in https://github.com/near/near-sdk-js/pull/337
* Update README.md by @bucanero in https://github.com/near/near-sdk-js/pull/338
* Remove C warnings, when build with --verbose by @ailisp in https://github.com/near/near-sdk-js/pull/339
* Fix build script when there is space in project path by @ailisp in https://github.com/near/near-sdk-js/pull/340
* gas and storage benchmarks by @ailisp in https://github.com/near/near-sdk-js/pull/341
* CI linux tests were not run because of a wrong condition by @ailisp in https://github.com/near/near-sdk-js/pull/343
* FT by @volovyks in https://github.com/near/near-sdk-js/pull/331
* fix(docs): contrib guidelines by @maxhr in https://github.com/near/near-sdk-js/pull/342
* Add links to reference doc by @ailisp in https://github.com/near/near-sdk-js/pull/344
* Remove alpha warning by @ailisp in https://github.com/near/near-sdk-js/pull/345
* Error and stop build a contract that has method name confliction by @ailisp in https://github.com/near/near-sdk-js/pull/346
* Rust specific terminology replaced with JS one by @volovyks in https://github.com/near/near-sdk-js/pull/348

## New Contributors
* @bucanero made their first contribution in https://github.com/near/near-sdk-js/pull/338

**Full Changelog**: https://github.com/near/near-sdk-js/compare/v0.7.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/near/near-sdk-js/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by kasodon** [comment]
Woot Woot 🎉
